### PR TITLE
Added github actions

### PR DESCRIPTION
**Pull request by sigmunsl**
Circle CI didn't give enough memory to build rasputin. Changed to Github Actions to make everything work properly.

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    container:
+      image: dnumgis/rasputin:latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Copy dependancies
```
Comment (stianlagstad) [inline .github/workflows/main.yaml]: dependancies -> dependencies

Comment (sigmunsl) [reply .github/workflows/main.yaml]: Fixed

**Comment by stianlagstad** [comment]
At this point do you just want to see if the build succeeds?

**Review by jakobes — APPROVED** [review]
Nice work. It looks like circle-ci is still running though.

**Comment by sigmunsl** [comment]
> At this point do you just want to see if the build succeeds?

Well, yeah. I've not implemented other tests yet. Focused on getting the testing up and running before implementing more tests.

**Review by stianlagstad — APPROVED** [review]

**Comment by sigmunsl** [comment]
> Nice work. It looks like circle-ci is still running though.

Yes. I struggled with removing it. By removing the config file, there is no longer anything that can run (I think?).

**Pull request merged**